packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -206,16 +206,18 @@ class BlockListBlock extends Component {
 				<View accessibilityLabel={ accessibilityLabel }>
 					{ isSelected && (
 						<FloatingToolbar>
-							<Toolbar passedStyle={ styles.toolbar }>
-								<ToolbarButton
-									title={ __( 'Navigate Up' ) }
-									onClick={ () =>
-										this.props.onSelect( parentId )
-									}
-									icon={ NavigateUpSVG }
-								/>
-								<View style={ styles.pipe } />
-							</Toolbar>
+							{ this.props.hasParent && (
```

Comment:
I suggest adding `hasParent` to the destructuring assignment like the rest of the props.

Reply:
done !